src/databricks/labs/ucx/hive_metastore/table_migrate.py

```diff
@@ -367,3 +368,85 @@ def _try_fetch(self) -> Iterable[MigrationStatus]:
     def _iter_schemas(self):
         for catalog in self._ws.catalogs.list():
             yield from self._ws.schemas.list(catalog_name=catalog.name)
+
+
+class TablesInMountsMigrator:
+    def __init__(
+        self,
+        tables_in_mounts: TablesInMounts,
+        ws: WorkspaceClient,
+        backend: SqlBackend,
+        table_mapping: TableMapping,
+    ):
+        self._backend = backend
+        self._ws = ws
+        self._tm = table_mapping
+        self._tim = tables_in_mounts
+
+    def create_tables_in_uc(self):
```

Comment:
How would this work with source code migration?

Reply:
It won't, and this PR has never been designed to.
Everything here has been made separatly because the TableMigrator class highly assumes that a table can be identified by a database.table, which is wrong in the case of a table in a mount point. A table in a mount point is identified by a path, it's used like this in customer codebases (same as external tables tho), and is mapped from path to catalog.database.table.

Because of this we either have to highly tweak the TableMigrator to include the special case of tables in a mount point, or treat this special case separatly.

Additionally, if tables in mounts needs to be compatible with the code migration,  we'll have to revamp the TableInMountCrawler a bit